projects/gui/src/mainwindow.cpp

```diff
@@ -725,6 +725,20 @@ void MainWindow::onGameFinished(ChessGame* game)
 			m_game = nullptr;
 		updateWindowTitle();
 	}
+
+	// save game notation of non-tournament games to default PGN file
+	if (!tab.m_tournament
+	&&  !game->pgn()->isNull()
+	&&  	(  !game->pgn()->moves().isEmpty()   // ignore empty games
+		|| !game->pgn()->result().isNone())) // without adjudication
+	{
+		QString fileName = QSettings().value("pgn/default_pgn_file", QString())
```

Comment:
I was actually planning on using the `pgn` namespace for PGN tags etc. I would rename this setting to `games/default_pgn_output_file` etc. I'm not sure what the namespace for non-tournament games should be, I guess `games` or `single_games` is okay. And there's already a `tournament` namespace so `pgn/default_tournament_pgn_file` would become `tournament/default_pgn_output_file` etc. I'd add the "output" part because games and tournament may also use PGN files as opening books (input).